modules/coo-troubleshooting-ui-plugin-using.adoc

```diff
@@ -0,0 +1,82 @@
+====
++
+image::coo-troubleshooting-panel-graph.png[Troubleshooting panel]
+[arabic]
+.. **Alert(1):** This node is the starting point in the graph and represents the `KubeContainerWaiting` alert displayed in the web console.
+
+.. **Pod(1):** This node indicates that there is a single `Pod` resource associated with this alert. Clicking on this node will open a console search showing the related pod directly.
+
+.. **Event(2):** There are two Kuberenetes events associated with the pod. Click this node to see the events.
+
+.. **Logs(74):** This pod has 74 lines of logs, which you can access by clicking on this node.
+
+.. **Metrics(105):** There are many metrics associated with the pod.
+
+.. **Network(6):** There are network events, meaning the pod has communicated over the network. The remaining nodes in the graph represent the `Service`, `Deployment` and `DaemonSet` resources that the pod has communicated with.
+
+.. **Focus:** Clicking this button updates the graph. By default, the graph itself does not change when you click on nodes in the graph. Instead, the main web console page changes, and you can then navigate to other resources using links on the page, while the troubleshooting panel itself stays open and unchanged. To force an update to the graph in the troubleshooting panel, click **Focus**. This draws a new graph, using the current resource in the web console as the starting point.
+
+.. **Show Query:** Clicking this button enables some experimental features:
++
+image::coo-troubleshooting-experimental.png[Experimental features]
+[arabic]
+... **Hide Query** hides the experimental features.
+
+... The query that identifies the starting point for the graph. 
+The query language, part of the link:https://korrel8r.github.io/korrel8r[Korrel8r] correlation engine used to create the graphs, is experimental and may change in future. 
+The query is updated by the **Focus** button to correspond to the resources in the main web console window. 
+
+... **Neighbourhood depth** is used to display a smaller or larger neighbourhood. 
++
+[NOTE]
+====
+Setting a large value in a large cluster may cause the query to fail, if the number of results is too big.
+====
+... **Goal class** results in a goal directed search instead of a neighbourhood search. A goal directed search shows all paths from the starting point to the goal class, which indicates a type of resource or signal. The format of the goal class is experimental and may change. Currently valid goals are:
+**** `k8s:__RESOURCE[VERSION.[GROUP]]__` identifies a kind of kuberenetes resource. For example `k8s:Pod` or `k8s:Deployment.apps.v1`.
+**** `alert:alert` for any alert.
+
+**** `metric:metric` for any metric.
+
+**** `netflow:network` for any network observability network event.
+
+**** `log:__LOG_TYPE__` for stored logs, where `__LOG_TYPE__` must be one of `application`, `infrastructure` or `audit`.
```

Comment:
Nit: These objects should have a noun of some type following them for translation reasons. I know you are under a tight deadline and that would require a refactoring this content. I don't think it is that big of a deal right now. Feel free to come back to this suggestion at another time.